examples/plot_dataset_example.py

```diff
@@ -64,7 +63,7 @@
 # a target y, and window_ind (which itself contains `i_window_in_trial`,
 # `i_start_in_trial`, and `i_stop_in_trial`, which are required for combining
 # window predictions in the scorer).
-for x, y, window_ind in windows_ds:
+for x, y, window_ind in windows_dataset:
```

Comment:
Feels weird that two `datasets` do not have the same API by default.
I think it would be nice to have an option that makes `windows_dataset` output `window_ind` but that is `False` by default no?

Reply:
what other dataset do you mean?

Reply:
The non-window one, for instance `MOABBDataset` or the one from `pytorch`.
I feel like using the `window_ind` is for specific use cases and thus should only be return when the user request it.

But maybe I do not understand completely this part, happy to discuss if you want.